web/src/components/molecules/Asset/Asset/AssetBody/preview-type-select.tsx

```diff
@@ -0,0 +1,45 @@
+import Select from "@reearth-cms/components/atoms/Select";
+import { DefaultOptionType } from "antd/lib/select";
+import { CSSProperties } from "react";
+
+export enum PreviewType {
+  IMAGE = "image",
+  GEO = "geo",
+  GEO3D = "geo3d",
+  MODEL3D = "model3d",
+  SVG = "svg",
+  ZIP = "zip",
+}
+
+type PreviewTypeSelectProps = {
+  onTypeChange: (
+    value: PreviewType,
+    option: DefaultOptionType | DefaultOptionType[]
+  ) => void | undefined;
+  style?: CSSProperties;
+  value?: PreviewType;
+};
+
+export const PreviewTypeSelect: React.FC<PreviewTypeSelectProps> = ({
+  onTypeChange,
+  style,
+  value,
+}) => {
+  const previewTypeList = [
+    { id: 1, name: "image", value: PreviewType.IMAGE },
+    { id: 2, name: "geo", value: PreviewType.GEO },
+    { id: 3, name: "geo3d", value: PreviewType.GEO3D },
+    { id: 4, name: "model3d", value: PreviewType.MODEL3D },
+    { id: 5, name: "svg", value: PreviewType.SVG },
+    { id: 5, name: "zip", value: PreviewType.ZIP },
```

Comment:
Are name and value not the same? I think you can combine them to just have {id: 1, type: PreviewType.Image}, etc

Reply:
The name should be what is displayed in the <Select /> element, while the value matches values in the PreviewType enum.
Name does not necessarily have to match value, please check the figma design.

<img width="446" alt="Screen Shot 0004-07-29 at 1 08 02 AM" src="https://user-images.githubusercontent.com/35835415/181590825-d16741e3-ffc1-44a9-8067-45d499035de2.png">

Reply:
<img width="616" alt="Screen Shot 0004-07-29 at 1 51 20 AM" src="https://user-images.githubusercontent.com/35835415/181594210-7da6a541-11ee-42b0-837d-a466492f3db8.png">

Reply:
But since you are using enums, won't PreviewType.IMAGE assign the value of PreviewType.IMAGE to the value here, which is "image"? Which means name is "image" and value is "image", no?

Reply:
Let's assume that we want to select "geo", at the same time we want to display "GEOJSON/KML/CZML" in the dropdown. 
In this case, name would be "GEOJSON/KML/CZML" and value would be "geo".
what I am saying is that name can be more descriptive than value.